app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1789,3 +1794,24 @@ fun MetadataCard(post: Post) {
         )
     }
 }
+
+@Composable
+fun TorrentHelpInfo() {
+    OutlinedCard(
+        shape = MaterialTheme.shapes.medium,
+        modifier =
+            Modifier
+                .padding(vertical = MEDIUM_PADDING, horizontal = MEDIUM_PADDING)
```

Comment:
Isn't this the same as `padding(MEDIUM_PADDING)`

Reply:
Good call.